docs/mkdocs/mkdocs.yml

```diff
@@ -17,6 +17,12 @@ theme:
 extra:
   version: 2.0
 nav:
+  - 'Project Information':
```

Comment:
@eriknordmark , now you have some yetus error here: 

`::group::docs/mkdocs/mkdocs.yml:::error: file=docs/mkdocs/mkdocs.yml,line=21,col=5::21:5: [indentation] wrong indentation: expected 6 but found 4`